quotesify.log

```diff
@@ -88,28 +88,28 @@ Oct 14, 2020 6:23:48 PM seedu.duke.commands.AddCommand execute
 INFO: going to add book to booklist
 Oct 14, 2020 6:23:48 PM seedu.duke.commands.AddCommand execute
 INFO: added book to booklist
-10�� 14, 2020 8:41:23 ���� seedu.duke.commands.AddCommand addToDo
-��Ϣ: add toDo to toDoList failed
-10�� 14, 2020 8:41:42 ���� seedu.duke.commands.AddCommand addToDo
-��Ϣ: add toDo to toDoList failed
-10�� 14, 2020 8:41:47 ���� seedu.duke.commands.AddCommand addToDo
-��Ϣ: add toDo to toDoList failed
-10�� 14, 2020 8:41:52 ���� seedu.duke.commands.AddCommand addToDo
-��Ϣ: add toDo to toDoList failed
-10�� 14, 2020 9:00:21 ���� seedu.duke.commands.AddCommand addToDo
-��Ϣ: add toDo to toDoList failed
-10�� 14, 2020 11:08:32 ���� seedu.quotesify.commands.AddCommand execute
-��Ϣ: going to add book to booklist
-10�� 14, 2020 11:08:32 ���� seedu.quotesify.commands.AddCommand execute
-��Ϣ: added book to booklist
-10�� 14, 2020 11:21:43 ���� seedu.quotesify.commands.AddCommand execute
-��Ϣ: going to add book to booklist
-10�� 14, 2020 11:21:43 ���� seedu.quotesify.commands.AddCommand execute
-��Ϣ: added book to booklist
-10�� 14, 2020 11:21:58 ���� seedu.quotesify.commands.AddCommand execute
-��Ϣ: going to add book to booklist
-10�� 14, 2020 11:21:58 ���� seedu.quotesify.commands.AddCommand execute
-��Ϣ: added book to booklist
+10月 14, 2020 8:41:23 下午 seedu.duke.commands.AddCommand addToDo
+信息: add toDo to toDoList failed
+10月 14, 2020 8:41:42 下午 seedu.duke.commands.AddCommand addToDo
+信息: add toDo to toDoList failed
+10月 14, 2020 8:41:47 下午 seedu.duke.commands.AddCommand addToDo
+信息: add toDo to toDoList failed
+10月 14, 2020 8:41:52 下午 seedu.duke.commands.AddCommand addToDo
+信息: add toDo to toDoList failed
+10月 14, 2020 9:00:21 下午 seedu.duke.commands.AddCommand addToDo
+信息: add toDo to toDoList failed
+10月 14, 2020 11:08:32 下午 seedu.quotesify.commands.AddCommand execute
+信息: going to add book to booklist
+10月 14, 2020 11:08:32 下午 seedu.quotesify.commands.AddCommand execute
+信息: added book to booklist
+10月 14, 2020 11:21:43 下午 seedu.quotesify.commands.AddCommand execute
+信息: going to add book to booklist
+10月 14, 2020 11:21:43 下午 seedu.quotesify.commands.AddCommand execute
+信息: added book to booklist
+10月 14, 2020 11:21:58 下午 seedu.quotesify.commands.AddCommand execute
+信息: going to add book to booklist
+10月 14, 2020 11:21:58 下午 seedu.quotesify.commands.AddCommand execute
+信息: added book to booklist
```

Comment:
Is the chinese because of some settings? :)

Reply:
i think this could be previously from Yilun's one